src/test/ui/nll/closure-requirements/region-lbr-anon-does-not-outlive-static.stderr

```diff
@@ -1,16 +1,21 @@
 warning: not reporting region error due to nll
-  --> $DIR/region-lbr-anon-does-not-outlive-static.rs:19:5
+  --> $DIR/region-lbr-anon-does-not-outlive-static.rs:20:5
    |
 LL |     &*x
    |     ^^^
 
 error[E0621]: explicit lifetime required in the type of `x`
-  --> $DIR/region-lbr-anon-does-not-outlive-static.rs:19:5
+  --> $DIR/region-lbr-anon-does-not-outlive-static.rs:18:1
    |
-LL | fn foo(x: &u32) -> &'static u32 {
-   |        - consider changing the type of `x` to `&ReStatic u32`
-LL |     &*x
-   |     ^^^ lifetime `ReStatic` required
+LL |   fn foo(x: &u32) -> &'static u32 {
+   |   ^      - consider changing the type of `x` to `&ReStatic u32`
```

Comment:
(this sort of thing is the reason I'm not really thrilled with `ui/` tests that use `-Z verbose` ... though I guess seeing `ReStatic` in supposed user output is not as egregious as other potential leakage of internal details...)